src/Sylius/Bundle/CoreBundle/SyliusCoreBundle.php

```diff
@@ -61,6 +63,8 @@ public function boot(): void
     {
         parent::boot();
 
+        $this->setSqlWalker();
+
```

Comment:
Why not a compiler pass?

Reply:
It has to be done in the boot method. I tried with a compiler pass but it didn't work.